sdk/cosmos/azure-cosmos/samples/test_xq.py

```diff
@@ -0,0 +1,82 @@
+import time
+import random
+from azure.cosmos.cosmos_client import CosmosClient as SyncClient
+from azure.cosmos.partition_key import PartitionKey
+
+endpoint = ''
+key = ''
+DATABASE_ID = 'dbid'
+CONTAINER_ID = 'containerid'
+
+import uuid
+
+
+def get_test_item():
+    async_item = {
+        'id': 'Async_' + str(uuid.uuid4()),
+        'address': {
+            'state': 'WA',
+            'city': 'Redmond',
+            'street': '1 Microsoft Way'
+        },
+        'test_object': True,
+        'lastName': 'Smith',
+        'attr1': random.randint(0, 100)
+    }
+    return async_item
+
+
+def partition_split_test():
+    client = SyncClient(endpoint, key)
+    db = client.create_database_if_not_exists(DATABASE_ID)
+    container = db.create_container_if_not_exists(id=CONTAINER_ID, partition_key=PartitionKey(path="/id"))
+    print("creating items")
+    start = time.time()
+    for i in range(15000):
+        body = get_test_item()
+        container.create_item(body=body)
+        if i == 5000 or i == 10000:
+            print("created 5k items")
+    print("15k items took {} seconds".format(time.time() - start))
+    success, errors = 0, 0
+    print("--------------------------------")
+    print("created items, waiting 4m before queries")
+    print("change offer to 11k manually")
+    print("--------------------------------")
+    time.sleep(240)
```

Comment:
Current test will not guarantee that your split completed , please refer https://github.com/Azure/azure-sdk-for-java/blob/c4c068cc0d2061df9a4fcc49419acc10be223d4a/sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/QueryValidationTests.java#L397